src/components/display/components/StickyToggleButtonComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react'
+import { StickyToggleButtonArgs } from 'display-api'
+import { ButtonComponent } from './ButtonComponent'
+
+export const StickyToggleButtonComponent = ({ button, name, on_change, default_value }: StickyToggleButtonArgs) => {
+	const [isEnabled, setEnabled] = useState<boolean>(default_value)
+
+	const onChange = () => {
+		setEnabled(!isEnabled)
+	}
+
```

Comment:
This needs to operate like a toggle, so when the value changes, it needs to set the variable for display API methods, like an input does when the value changes.